app/assets/javascripts/admin/addon/components/bulk-user-delete-confirmation.gjs

```diff
@@ -24,15 +25,17 @@ export default class BulkUserDeleteConfirmation extends Component {
   callAfterBulkDelete = false;
   blockIpAndEmail = false;
 
-  constructor() {
-    super(...arguments);
-    this.messageBus.subscribe(BULK_DELETE_CHANNEL, this.onDeleteProgress);
-  }
+  logsListener = modifierFn(() => {
+    this.messageBus.subscribe(
+      BULK_DELETE_CHANNEL,
+      this.onDeleteProgress,
+      this.args.model.lastBulkDeleteMessageBusId
```

Comment:
These changes were made to make this spec more reliable it was flakey because of this missing id.